docs/_docs/shell/connect.md

```diff
@@ -198,14 +116,13 @@ mqtt>
 
 ***
 
-> Connect a client which is identified by myClient on specific settings and disconnect it afterwards
+Connect a client which is identified by myClient on specific settings and disconnect it afterward
```

Comment:
I'd stick with afterwards, but both seem correct

Reply:
Yeah also way more natural to me, but since all of our documentation is written in AE and IntelliJ also suggests using it, I changed it.